js/ascendex.js

```diff
@@ -257,6 +257,38 @@ module.exports = class ascendex extends Exchange {
                 'transfer': {
                     'fillResponseFromRequest': true,
                 },
+                'defaultNetworks': {
```

Comment:
what is the point of having this `defaultNetworks` dict?

Reply:
@carlosmiei I don't remind where we I had discussed in the past (we had I remember) but one related topic is here: https://discord.com/channels/690203284119617602/996342978236915753

So, these `defaultNetwork(s)` props are just for the case when user doesn't provide the `networkCode` in a method, that has some interaction with chains (i.e. fetchDepositAddress, withdraw, ...) they do need `networkCode`, but in case user doesn't provide that value, then exchange should check, which network should be chosen for that coin. I.e. in defaultNetworks you can write:
```{
   ETH: ERC20,
   USDT: TRC20,
}```
so, user making `withdraw('USDT', address)` will resolve to that default network which is set to TRC20. till date, there is mostly used `defaultNetwork` exchange wide singular property, which does resolve for "all" coins, but the plural version of that property (`defaultNetworks`) is more flexible.

 Bth, I voted for their inclusion in signature (instead of `params[network]`) and might even make it mandatory, to avoid any "defaults" (which might theoretically be against user's expectations)

Reply:
@ttodua the thing is I don't see any reference to it, so it's not being used right?

Reply:
I've removed it only that part now. the PR should be ok now.